includes/classes/ExternalConnections/WordPressExternalConnection.php

```diff
@@ -556,7 +559,7 @@ public function push( $post_id, $args = array() ) {
 			'slug'                           => $post->post_name,
 			'content'                        => Utils\get_processed_content( $post->post_content ),
 			'type'                           => $post->post_type,
-			'status'                         => ( ! empty( $args['post_status'] ) ) ? $args['post_status'] : 'publish',
+			'status'                         => ( ! empty( $args['post_status'] ) ) ? $args['post_status'] : $distribute_status,
```

Comment:
@adamsilverstein If I mark `$distribute_post_status` as true, the source post status will only be distributed if `$args['post_status']` is empty. `$args['post_status']` comes from the connection map for the origin post, how is that populated or changed? In any case, going off the filter name I would expect `$distribute_post_status` to always take precedence, but it's very possible I'm woefully missing context here.

Reply:
In this case of push, the `post_status` comes from whether you have checked the 'distribute as draft' checkbox, if you have that setting overrides the filter. I do need to test subscriptions and pull, i'm not sure that works correctly. See https://github.com/10up/distributor/blob/b1dd60f9fdb6e22efd84a85807a6b7aec91e615e/includes/push-ui.php#L248-L250